pages/curriculum.tsx

```diff
@@ -48,11 +49,12 @@ export const Curriculum: React.FC<GetAppProps> = ({ data }) => {
   const lessonsToRender: React.ReactElement[] = lessons.map((lesson, idx) => {
     const id = _.get(lesson, 'id', idx) as number
     const status = lessonStatusMap[id]
+    const passed = _.get(status, 'isPassed', false)
     let lessonState = ''
     if (idx === lessonInProgressIdx) {
       lessonState = 'inProgress'
     }
-    if (status?.isPassed) {
```

Comment:
put variables declarations close to where it is used. so move `const passed = ....` to here.